wallet/androidTest/de/schildbach/wallet/performance/WalletSavePerformanceTest.kt

```
@@ -0,0 +1,281 @@
+                log.debug("Skipped invalid test transaction $i: ${e.message}")
+            }
+        }
+        
+        log.info("Successfully added transactions. Wallet now has ${wallet.getTransactionCount(true)} transactions")
+    }
+
+    private fun createTestTransactionData(index: Int): ByteArray {
+        // Create a realistic transaction structure for testing
+        // This is a simplified version - in reality you'd want more varied transaction types
+        val baseHex = "01000000013511fbb91663e90da67107e1510521440a9bf73878e45549ac169c7cd30c826e"
+        val indexHex = String.format("%02d", index % 256)
```

detekt (1.23.8)

[warning] String.format("%02d", index % 256) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)